From c72927ae2c65403c58639d55388fcfbec047a035 Mon Sep 17 00:00:00 2001
From: Joseph Fernandes <josferna@redhat.com>
Date: Wed, 6 Jan 2016 20:17:28 +0530
Subject: [PATCH 229/237] tier/glusterd : Corrected default values for sql cache and wal

Corrected default values for sql cache and wal

Change-Id: I5e659e6c9501e131c570f01fb9c20493ab092d7f
BUG: 1288509
Signed-off-by: Joseph Fernandes <josferna@redhat.com>
Reviewed-on: https://code.engineering.redhat.com/gerrit/64971
Reviewed-by: Dan Lambright <dlambrig@redhat.com>
Tested-by: Dan Lambright <dlambrig@redhat.com>
---
 xlators/mgmt/glusterd/src/glusterd-volume-set.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/xlators/mgmt/glusterd/src/glusterd-volume-set.c b/xlators/mgmt/glusterd/src/glusterd-volume-set.c
index 2babd2f..da1f15a 100644
--- a/xlators/mgmt/glusterd/src/glusterd-volume-set.c
+++ b/xlators/mgmt/glusterd/src/glusterd-volume-set.c
@@ -2462,7 +2462,7 @@ struct volopt_map_entry glusterd_volopt_map[] = {
         },
         { .key         = "features.ctr-sql-db-cachesize",
           .voltype     = "features/change/timerecorder",
-          .value       = "1000",
+          .value       = "12500",
           .option      = "sql-db-cachesize",
           .validate_fn = validate_ctr_sql_params,
           .op_version  = GD_OP_VERSION_3_7_6,
@@ -2476,7 +2476,7 @@ struct volopt_map_entry glusterd_volopt_map[] = {
         },
         { .key         = "features.ctr-sql-db-wal-autocheckpoint",
           .voltype     = "features/changetimerecorder",
-          .value       = "1000",
+          .value       = "25000",
           .option      = "sql-db-wal-autocheckpoint",
           .validate_fn = validate_ctr_sql_params,
           .op_version  = GD_OP_VERSION_3_7_6,
-- 
1.7.1

